provider/terraform.rb

```diff
@@ -179,6 +180,7 @@ def generate_resource_tests(data)
       )
 
       # TODO: error check goimports
+      %x(gofmt -w -s #{filepath})
```

Comment:
Since `goimports` does 90% of what gofmt does this introduces a lot of io churn for not that much. Why not just change the templating code to remove things like superfluous references to `schema.Schema` instead of putting it through 2 formatting cycles?

Reply:
Imo this solves the problem in the future if new harder to follow rules are added to gofmt -s, and takes almost no time during MM generation, so it's not worth spending time doing it by hand.